shared/locales/en/feedback.json

```diff
@@ -0,0 +1,54 @@
+{
+    "smartling": {
+        "string_format" : "icu",
+        "translate_paths": [{
+            "path": "*/title",
+            "key": "{*}/title",
+            "instruction": "*/note"
+        }]
+    },
+    "brokenSiteLabel": {
+        "title": "Which website is broken?",
+        "note": "Input label for broken site URL"
+    },
+    "brokenSitePlaceholder": {
+        "title": "Copy and paste your URL",
+        "note": "Placeholder text on URL input"
+    },
+    "feedbackHeaderLabel": {
+        "title": "What do you love? What isn't working? How could the extension be improved?",
+        "note": "Label over feedback textarea"
```

Comment:
"Label over a feedback text area where the user can tell us about their experience with our browser extension."